src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -2023,7 +2023,7 @@ def aead_cipher_supported(self, cipher) -> bool:
         return self._lib.EVP_get_cipherbyname(cipher_name) != self._ffi.NULL
 
     @contextlib.contextmanager
-    def _zeroed_bytearray(self, length):
+    def _zeroed_bytearray(self, length: int) -> typing.Iterator[bytearray]:
```

Comment:
I don't understand how typing works with `contextlib.contextmanager`. I guess this accurately describes the inner function?

Reply:
I guess so? I just tried to crib out of typeshed here honestly